Dockerfile

```diff
@@ -42,9 +42,8 @@ COPY ./src/ununpack/mod_deps ./src/ununpack/
 COPY ./src/wget_agent/mod_deps ./src/wget_agent/
 
 RUN mkdir -p /fossology/dependencies-for-runtime \
- && cp -R /fossology/src /fossology/utils /fossology/dependencies-for-runtime/
-
-RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+ && cp -R /fossology/src /fossology/utils /fossology/dependencies-for-runtime \
+ && DEBIAN_FRONTEND=noninteractive apt-get update \
```

Comment:
As a standard best practice, the installation of dependencies should always be done in a separate layer. I do not see any advantage of this change here.